lib/features/home/domain/extensions/session_extensions.dart

```diff
@@ -43,4 +45,14 @@ extension SessionExtensions on Session {
       return apiUrl.toString();
     }
   }
+
+  String get serverDomain {
+    if (GetUtils.isEmail(username.value)) {
+      return username.value.split('@').last;
+    } else if (GetUtils.isEmail(personalAccount.name.value)) {
```

Comment:
`personalAccount` getter maybe throw exception, please handle this case also

Reply:
writing test for this function also

Reply:
done